File: A1/A1-013.cpp
#include <iostream>
using namespace std;

int main()
{
    int pn;
    char pc[2];

    // input
    cin >>pc >>pn;

    // process
    if (pc[0] == 'H' && pn == 4567) cout<< "safe unlocked";
    else if (pc[0] != 'H' && pn == 4567) cout<< "safe locked - change char";
    else if (pc[0] == 'H' && pn != 4567) cout<< "safe locked - change digit";
    else cout<< "safe locked";   
}

File: Python/A1/A1-013.py
# Author: Korawit Suporn
# Date  : 23 Apr. 2025
# Editor: Vim

ch = input()
ps = input()

collection = ['H', "4567"]

if ch in collection and ps in collection:
    print("safe unlocked")
elif ps in collection:
    print("safe locked - change char")
elif ch in collection:
    print("safe locked - change digit")
else:
    print("safe locked")
